shader/flatten.wgsl

```diff
@@ -407,55 +488,49 @@ fn main(
     if seg_type != 0u {
         let is_stroke = (style_flags & STYLE_FLAGS_STYLE_BIT) != 0u;
         let transform = read_transform(config.transform_base, trans_ix);
-        let pts = read_path_segment(tag, transform, is_stroke);
-        var bbox = vec4(min(pts.p0, pts.p1), max(pts.p0, pts.p1));
-        bbox = vec4(min(bbox.xy, pts.p2), max(bbox.zw, pts.p2));
-        bbox = vec4(min(bbox.xy, pts.p3), max(bbox.zw, pts.p3));
+        let pts = read_path_segment(tag, is_stroke);
 
         var stroke = vec2(0.0, 0.0);
         if is_stroke {
             let linewidth = bitcast<f32>(scene[config.style_base + style_ix + 1u]);
+            let offset = 0.5 * linewidth;
+
             // See https://www.iquilezles.org/www/articles/ellipses/ellipses.htm
             // This is the correct bounding box, but we're not handling rendering
             // in the isotropic case, so it may mismatch.
-            stroke = 0.5 * linewidth * vec2(length(transform.mat.xz), length(transform.mat.yw));
-            bbox += vec4(-stroke, stroke);
+            stroke = offset * vec2(length(transform.mat.xz), length(transform.mat.yw));
```

Comment:
`stroke` is now unused. I've been thinking about removing this field from the `cubic` struct or possibly replacing `Cubic` with `CubicPoints` + just the `path_ix`.